spec/features/admin/admin_can_create_a_tutorial_spec.rb

```diff
@@ -45,6 +45,7 @@
           click_on "Save"
 
           expect(page).to_not have_content("Successfully created tutorial.")
+          expect(page).to have_content("Unable to create a tutorial.")
```

Comment:
Didn't make more specific errors for the tutorial but I could, if you think it's a good idea.

Reply:
I did a little bit. We can revisit. Its more complicated now.